shell/platform/windows/window.cc

```diff
@@ -648,4 +661,8 @@ bool Window::GetHighContrastEnabled() {
   }
 }
 
+void Window::CreateAccessibilityRootNode() {
+  accessibility_root_ = AccessibilityRootNode::Create();
```

Comment:
Looks like we're careful to currently only ever call this if accessibility_root_ is null, but might be a good idea to defensively checking this for null and release the old one if non-null.